xml/System.Buffers/IPinnable.xml

```diff
@@ -40,10 +40,19 @@
         <Parameter Name="elementIndex" Type="System.Int32" />
       </Parameters>
       <Docs>
-        <param name="elementIndex">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="elementIndex">The offset to the element within the memory buffer to which the returned <see cref="T:System.Buffers.MemoryHandle" /> points.</param>
+        <summary>Pins a block of memory.</summary>
+        <returns>A handle to the block of memory.</returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+
+Call this method to indicate that the <xref:System.Buffers.IPinnable> object cannot be moved by the garbage collector so that the address of the pinned object can be used.
```

Comment:
I like this, but I'm wondering if it could be better. 

A developer can access the object without pinning it, but only through managed APIs. While pinned, unmanaged techniques can be used.